build.gradle

```diff
@@ -18,14 +22,24 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'org.omegat:vldocking:3.0.5'
+    compileOnly 'org.omegat:vldocking:3.0.5'
+    compileOnly 'org.omegat:omegat:5.2.0'
     testCompile group: 'junit', name: 'junit', version: '4.11'
 }
 
+javafx {
+    version = '12'
+    modules = [ 'javafx.swing', 'javafx.web']
+    configuration = 'compile'
```

Comment:
When specified here as "compile" not "compleOnly", platform dependent libraries are also bundled with package fat-jar.
Maintainer should build packages on each platform they want to support.